### PR TITLE
[WIP] Merged Installation and Upgrade guides to single book

### PR DESCRIPTION
**Pull request by 0rnela**
## Please do not approve yet.

# Description

Installation and Upgrade guides have been merged. Conceptually, the topics are of similar kind. The new single book also adds to tidier look&feel of the navigation bar.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

This PR replaces the work initially started in https://github.com/uyuni-project/uyuni-docs/pull/1329, which has been broken down to smaller segments due to the volume of changes that were created.

### CODE REVIEW

**Review by keichwa — APPROVED** [review]

File: modules/installation-and-upgrade/pages/install-overview.adoc_backup [hunk 1]
```diff
@@ -1,14 +1,16 @@
-[[installation-overview]]
-= Installation Guide Overview
+[[installation-and-upgrade-overview]]
+= TESTING TESTING ARE YOU SERIOUS Installation and Upgrade Guide Overview
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/install-overview.adoc_backup:2]: Is this intentional? :D

File: modules/installation-and-upgrade/pages/installation-and-upgrade-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+= Installation and Upgrade Guide Overview
+
+**Updated:** {docdate}
+
+This book provides guidance on installing and upgrading {productname} Server and Proxy.
+It is split into four sections:
+
+* Requirements: Describes the hardware, software, and networking requirements that you require before you begin.
+* Installation: Describes the process to install {productname} components.
+* Setting Up: Describes the initial steps you need to take after installation to make your {productname} environment ready to use.
```
Comment (jcayouette) [inline modules/installation-and-upgrade/pages/installation-and-upgrade-overview.adoc:11]: Rename to "Setup"

**Review by jcayouette — CHANGES_REQUESTED** [review]
Keep in mind that moving around these files will break links to former guides. You will need to update these broken xrefs as they appear in your build. (unless you have done so already) Overall looks good 👍

File: modules/installation-and-upgrade/pages/installation-and-upgrade-overview.adoc (hunk 2 above)
Comment (0rnela) [reply modules/installation-and-upgrade/pages/installation-and-upgrade-overview.adoc:11]: @jcayouette - good catch, I can change it. However, "Setting Up" was actually inherited from previous versions.

File: modules/installation-and-upgrade/pages/install-overview.adoc_backup (hunk 1 above)
Comment (0rnela) [reply modules/installation-and-upgrade/pages/install-overview.adoc_backup]: LOL, testing testing ;)

**Review by keichwa — APPROVED** [review]
assuming that everything will work as it should.

File: modules/administration/pages/backup-restore.adoc [hunk 3]
```diff
@@ -73,9 +73,9 @@ You also need to back up your database, which you can do with the [command]``smd
 
 .Procedure: Restoring from a Manual Backup
 . Re-install the {productname} Server.
-  For more information, see xref:installation:install-server-unified.adoc[]].
+  For more information, see xref:installation-and-upgrade:install-server-unified.adoc[]].
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:76]:
```suggestion
  For more information, see xref:installation-and-upgrade:install-server-unified.adoc[].
```

**Review by keichwa — APPROVED** [review]

**Comment by 0rnela** [comment]
Superseeded by https://github.com/uyuni-project/uyuni-docs/pull/1346

**Pull request closed without merging**